test/gtest/uct/test_error_handling.cc

```diff
@@ -27,6 +27,10 @@ class test_error_handling : public uct_test {
             GetParam()->tl_name == "dc" || GetParam()->tl_name == "dc_mlx5") {
             set_config("RC_TIMEOUT=0.0001"); /* 100 us should be enough */
             set_config("RC_RETRY_COUNT=2");
+        } else if (GetParam()->tl_name == "ud") {
+            set_config("TIMEOUT=1s");
+        } else if (GetParam()->tl_name == "ud_mlx5") {
+            set_config("UD_TIMEOUT=1s");
```

Comment:
Because it does not work, some prefix magic in config API.
However, it looks equally in ucx_info output:
```
$ucx_info -c | grep TIMEOUT
UCX_UD_VERBS_TIMEOUT=60000000.00us
UCX_UD_MLX5_TIMEOUT=60000000.00us
...
```